### PR TITLE
Fixes #8252: Document how to connect to a non-default packager port o…

### LINKED ISSUE #8252: Document how to connect to a non-default packager port on Android
It's possible to start the packager locally on a different than the default port 8081:

    react-native start --port 8088

(Try going to localhost:8088 in your browser)

How to get the device to connect to the packager on that port, however?

On iOS, you can change the port in AppDelegate.m.
On Android, the port seems to be hardcoded in [AndroidInfoHelpers.java](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java), maybe it's possible to simply pass different arguments to `adb reverse tcp:8081 tcp:8081`?

A good place to put this info would be the "Running on device" docs, both iOS and Android:
- http://facebook.github.io/react-native/docs/running-on-device-ios.html
- http://facebook.github.io/react-native/docs/running-on-device-android.html

This is quite highly upvoted request on Product Pains:
https://productpains.com/post/react-native/allow-packager-port-to-be-configurable-change-from-8081/

### PR DESCRIPTION
**Pull request by burgalon**
Added some documentation to the `RunningOnDeviceAndroid.md` with screenshots to set custom port

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @mkonicek to be potential reviewers.

File: docs/RunningOnDeviceAndroid.md [hunk 1]
```diff
@@ -93,6 +93,22 @@ Have your device connected via USB with debugging enabled (see paragraph above o
 1. Run `adb reverse tcp:8081 tcp:8081`
 2. You can use `Reload JS` and other development options with no extra configuration
 
+
+### Running packager on a non standard port
+
+Launch the pacakager manually with `react-native start --port [PORT]`
```
Comment (mkonicek) [inline docs/RunningOnDeviceAndroid.md]: nit: packager

**Comment by mkonicek** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-0** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/284280575259616/int_phab) to review.

**Comment by mkonicek** [comment]
Reviewed in the Tel Aviv hackathon.

File: docs/RunningOnDeviceAndroid.md [hunk 2]
```diff
@@ -93,6 +93,22 @@ Have your device connected via USB with debugging enabled (see paragraph above o
...
+For Android: Use the developer menu by clicking the menu button or shake. Select 'Debug server host & port for device' to set a different port.
+<center>
+  <img src="img/AndroidDeveloperMenu.png" width="162" />
+  <img src="img/AndroidDevSettings.png" width="150" />
+  <img src="img/AndroidDevServerDialog.png" width="150" />
+</center>
+
+For IOS: Edit the AppDelegate.m and change the line below to match the port number you're running:
+
+```jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=true"];```
```
Comment (javache) [inline docs/RunningOnDeviceAndroid.md]: This is out-of-date with the new RCTBundleURLProvider API

Comment (burgalon) [inline docs/RunningOnDeviceAndroid.md]: So I guess I should just remove the IOS notes right?

**Pull request closed without merging**